hq_superset/tests/test_views.py

```diff
@@ -134,7 +134,7 @@ def _do_assert(datasources):
         client.get('/hq_datasource/list/', follow_redirects=True)
         _do_assert(self.oauth_mock.test2_datasources)
 
-    @patch.object(DomainSyncUtil, "sync_domain_role", return_value=True)
+    @patch.object(DomainSyncUtil, "_get_domain_access", return_value=({"can_write": True, "can_read": True}, []))
```

Comment:
It was a flaky test that succeeded when run separately or if only this test file was run but failed when the full test suite was run.
I believe the permissions were getting set unexpectedly or getting revoked somehow, so I set them specifically instead of skipping it via the patch for `sync_domain_role`. I did try to debug it for a couple of hours before letting it be.